### PR TITLE
[G2M] Plotly - fix custom axis title for plotly Bar chart; add custom axis titles for BarLine chart

### PR DESCRIPTION
**Pull request by geoerika**
**Changes:**
### Plotly/Bar

1. Fixed custom title for plotly Bar chart
2. Added story for horizontal bar chart with custom axis titles

Screen Shot 2022-10-12 at 12 46 15 PM

[Test Story](https://60f5a69eddb5b90039b6d17c-nheqlcgjdn.chromatic.com/?path=/story/plotly-bar--horizontal-custom-axis-titles)

### Plotly/BarLine

1. Added custom `x-axis` props for display & title text
Screen Shot 2022-10-12 at 10 16 07 AM

3. Added x-axis title & fix positioning of the title
**Before:**
Screen Shot 2022-10-12 at 10 07 14 AM

[Test Story](https://60f5a69eddb5b90039b6d17c-ebjgpqscuj.chromatic.com/?path=/story/plotly-bar-line--custom-axis-titles)

### CODE REVIEW

**Review by Hamaad102 — APPROVED** [review]
Looks good, there aren't any issues with positioning when reshaping the graph.

**Review by kc-leung — APPROVED** [review]

File: src/components/plotly/bar-line/index.js [hunk 1]
```diff
@@ -100,7 +108,8 @@ BarLine.propTypes = {
   x: PropTypes.string.isRequired,
   y: PropTypes.arrayOf(PropTypes.string).isRequired,
   showTicks: PropTypes.bool,
-  showAxisTitles: PropTypes.bool,
+  showAxisTitles: PropTypes.objectOf(PropTypes.bool),
+  axisTitles: PropTypes.objectOf(PropTypes.string),
```
Comment (woozyking) [inline src/components/plotly/bar-line/index.js:112]: maybe for next iteration we can use shape (exact) to make this a bit more rigid and self-documenting (VS Code does a good job inferring prop types for component usage doc):

```suggestion
  showAxisTitles: PropTypes.shape({
    x: PropTypes.bool,
    y: PropTypes.bool,
    y2: PropTypes.bool,
  }),
  axisTitles: PropTypes.shape({
    x: PropTypes.string,
    y: PropTypes.string,
    y2: PropTypes.string,
  }),
```

**Review by woozyking — APPROVED** [review]

File: src/components/plotly/bar-line/index.js (hunk 1 above)
Comment (geoerika) [reply src/components/plotly/bar-line/index.js:112]: @woozyking do you mind if I change it now here?

Comment (woozyking) [reply src/components/plotly/bar-line/index.js:112]: absolutely!

**Pull request merged**